tutorials/inputs/inputevent.rst

```diff
@@ -95,8 +95,8 @@ received input, in order:
    If any function consumes the event, it can call :ref:`SceneTree.set_input_as_handled() <class_SceneTree_method_set_input_as_handled>`, and the
    event will not spread any more. The unhandled input callback is ideal for full-screen gameplay events, so they are not received when a GUI is active.
 4. If no one wanted the event so far, and a :ref:`Camera <class_Camera>` is assigned
-   to the Viewport, a ray to the physics world (in the ray direction from
-   the click) will be cast. If this ray hits an object, it will call the
+   to the Viewport with :ref:`Object Picking <class_viewport_property_physics_object_picking>` turned on, a ray to the physics world (in the ray direction from
+   the click) will be cast. (For the root viewport, this can also be enabled in :ref:`Project Settings <class-projectsettings-property-physics-common-enable-object-picking>`) If this ray hits an object, it will call the
```

Comment:
```suggestion
   the click) will be cast. (For the root viewport, this can also be enabled in :ref:`Project Settings <class_ProjectSettings_property_physics/common/enable_object_picking>`) If this ray hits an object, it will call the
```